snap/snapcraft.yaml

```diff
@@ -1,52 +1,88 @@
-name: spark-k8s # you probably want to 'snapcraft register <name>'
+name: spark-client
 base: core20
 version: '0.1'
-summary: Client side snap to submit spark jobs to kubernetes.
+summary: Client side scripts to submit Spark jobs to a cluster.
 description: |
   Spark is now available on Kubernetes and needs a client to submit and 
   interact with jobs. This snap packages the client binaries for the same.
 
-grade: devel
+grade: stable
 confinement: strict
 
+plugs:
+  procsys-read:
+    interface: system-files
+    read:
+      - /proc/cgroups
+      - /proc/self/coredump_filter
```

Comment:
are these needed?

Reply:
Added these as suggested by errors in snappy-debug helper session. Let me figure out the bare-minimum needed for job submission to go through.